src/sentry/api/endpoints/organization_events_trace.py

```diff
@@ -562,8 +565,13 @@ def augment_transactions_with_spans(
         problem_project_map = {}
         issue_occurrences = []
         occurrence_spans = set()
-        error_spans = {e["trace.span"] for e in errors if e["trace.span"]}
-        projects = {e["project.id"] for e in errors if e["trace.span"]}
+        error_spans = set()
+        projects = set()
+        for error in errors:
+            if "trace.span" in error:
+                error["trace.span"] = pad_span_id(error["trace.span"])
+                error_spans.add(error["trace.span"])
+            projects.add(error["project.id"])
```

Comment:
we were also only adding project id if trace.span, so does this also need to be nested under the if condition

Reply:
No i think that was just bad copy paste on my part lol